administration/jsondb.md

```diff
@@ -96,7 +96,7 @@ Within the `OPENHAB_USERDATA/jsondb/` directory, you will find the following fil
 In this example, we will use the Network Binding (2.0) to Search for Things, add a new Thing to openHAB and then modify its parameters to check the information that is stored in the JsonDB.
 
 Step 1. Add new Thing (name: `ISP_Gateway`) from Paper UIs Inbox:
-![Add_Thing_Paper_UI](http://imageshack.com/a/img924/8255/jIl6pj.png)
+![Add_Thing_Paper_UI](images/add_thing_paper_ui.png)
```

Comment:
The proper way of making relative links on the website is:
`./images/...`

This syntax would have worked too though, because it's fixed by the prepare-docs website:

https://github.com/openhab/website/blob/743f0b867e4d26a5a6a3116a12f6344295ef25d6/prepare-docs.rb#L247

It's best to have it done right for new things though!

Reply:
Fully agree!